cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/AttachmentManager.java

```diff
@@ -47,7 +47,6 @@
  */
```

Comment:
Comment `Attachments are stored on disk in an extension directory, {@code EXTENSION_NAME}.` probably needs to change.